lib/src/ignore/files.rs

```diff
@@ -1,4 +1,6 @@
+use git_config::file::GitConfig;
```

Comment:
This should be `ResolvedGitConfig` as per the [docs](https://docs.rs/git-config/0.1.11/git_config/file/struct.GitConfig.html)